### DISCUSSION TITLE
with-styled-components example needed?

### DISCUSSION THREAD

**Discussion by hanselabreu** [vercel/next.js#16979] [Help] [answered]
I setup a project that is using `styled-components` and not using the recommended `_document.js` [file](https://github.com/vercel/next.js/blob/canary/examples/with-styled-components/pages/_document.js) in the `with-styled-components` example and it's running fine with no errors/warnings. Is doing all that still a requirement?

**Comment by kartikag01** [comment, marked as answer]
I think it is required when you want styles to work with SSR. Try disable JS in browser `styled-components` styles will not work without _document.js changes. 
@hanselabreu

**Reply by hanselabreu** [reply]
You are indeed correct, just tried it and it won't work with JS disabled.
